mica/starcheck/process.py

```diff
@@ -149,9 +151,11 @@ def update(config=None):
     else:
         db = Ska.DBI.DBI(dbi='sqlite', server=config['starcheck_db']['server'])
         max_mtime = db.fetchone("select max(mtime) as mtime from starcheck_id")
-        if max_mtime is not None:
+        if max_mtime is not None and 'start' not in config:
```

Comment:
Since it isn't computationally expensive, you can revert this change and just have the override at the end.  This simplifies the logic.